### ISSUE TITLE
Incompatible requirements error in components.ipynb on Colab

### ISSUE THREAD

**Issue by tall-josh** [tensorflow/tfx#1595]
Hi I'm trying to run the TFX components notebook. It ran without a problem about a month ago, but now I am met with this after running the first `!pip install ...` cell.

![image](https://user-images.githubusercontent.com/18511395/78662023-101e9280-7913-11ea-9469-edb3865ef8ab.png)

**Comment by gowthamkpr** [comment]
@tall-josh Just progress further ignoring those error and let me know if you are running into any errors after this. Thanks!

**Comment by tall-josh** [comment]
Thanks @gowthamkpr 

I just tried but I am unable to restart the runtime. When I try to restart the runtime the status icon goes from "Restarting" to "Initializing" then disconnects. Then when I attempt to reconnect it show "Busy" indefinitely. This has been occurring for the past week or so.

Also, the "Restart Runtime" button no longer appears after running the pip install cell like it used to, not sure if that's related either.

**Comment by gowthamkpr** [comment]
@tall-josh I was able to successfully install these packages 
`!pip install "tfx>=0.21.1,<0.22" "tensorflow>=2.1,<2.2" "tensorboard>=2.1,<2.3" "pyzmq==17.0.0"`
 and was able to restart the session on colab. 

Not sure why you are not able to do the same.

**Comment by tall-josh** [comment]
DUDE! Now I'm confused. I have been trying to run that cell for over a week and it kept exiting after the error messages shown above. Then not allowing me to restart the runtime. I ran the same cell again just then and it's working. I noticed a few differences in the output.

1. It's back to printing wheel url. Not just the progress bar. ![image](https://user-images.githubusercontent.com/18511395/78959441-4ab04700-7b2e-11ea-850e-5f8596cae82b.png)

2. The cell continues to execute after the errors. 
![image](https://user-images.githubusercontent.com/18511395/78959531-906d0f80-7b2e-11ea-947e-2562441b351d.png)

3. The Restart Runtime button is back. 
![image](https://user-images.githubusercontent.com/18511395/78959562-a67ad000-7b2e-11ea-8b73-47006a7fdae9.png)

Either way. It's working now. I'll make a closed. Thanks for your time.
